docs/site/Examples.md

```diff
@@ -64,6 +64,9 @@ LoopBack 4 comes with the following example projects.
   [Todo example](https://github.com/strongloop/loopback-next/tree/master/examples/todo)
   with JWT authentication.
 
+- **[build jwt authentication from scratch](https://github.com/HrithikMittal/Loopback4-auth)**:
```

Comment:
Just wondering if we should mention this is a community example? I know you mentioned in the other page that some are community examples, but if someone goes directly to this page they wouldn't see that note.

Reply:
If we're planning to have more community examples on this page, perhaps we can use superscript or asterisk to indicate this is community example?

Reply:
I realized this example page is for LB supported examples only...let me find move it to another place.

Reply:
I moved them back to be under the auth overview page. And use `(community)` to mark community examples.